pages/quizzes/html-quiz/html-quiz.html

```diff
@@ -0,0 +1,461 @@
+        </h3>
+
+        <div class="spacer-3rem"></div>
+        <div class="spacer-3rem"></div>
+        <div class="flex justify-between align-items-center">
+          <a
+            href="/pages/quizzes/html-quiz/html-quiz-result.html"
+            class="btn shadow-md btn-square-solid bg-cyan-500 btn-bg-hover-cyan-900 btn-text-hover-white"
+            >Check Score</a
+          >
+
+          <div id="modal" style="z-index: 30" class="modal">
+            <div class="modal-content bg-white">
+              <div
+                class="modal-header flex justify-between align-items-center bg-grey-800 text-white p-5"
+              >
+                <h4 class="text-lg">Quiz Rules</h4>
+                <span
+                  onclick="closeModal()"
+                  class="modal-close text-rose-500 text-hover-cyan-300"
+                  >×</span
+                >
+              </div>
+              <div class="modal-body p-5">
+                <p>Some rules in the Modal Body</p>
+                <p>Some other rules...</p>
+                <p>Some rules in the Modal Body</p>
+                <p>Some other rules...</p>
+                <p>Some rules in the Modal Body</p>
+                <p>Some other rules...</p>
+                <p>Some rules in the Modal Body</p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="spacer-3rem"></div>
+        <div class="spacer-3rem"></div>
+
+        <div class="question-wrapper">
+          <div class="flex justify-between align-items-center mb-10">
+            <div class="flex justify-center align-items-center">
+              <h6 class="mr-3 text-lg text-cyan-500 font-bold">Question:</h6>
+              <h6 class="text-lg font-normal">1/5</h6>
+            </div>
+
+            <div class="flex justify-center align-items-center">
+              <h6 class="mr-3 text-lg font-bold text-cyan-500">Score:</h6>
+              <h6 class="text-lg font-normal">0</h6>
+            </div>
+          </div>
+
+          <h5 class="text-3xl font-normal mb-10">
+            1. What does the abbreviation HTML stand for?
+          </h5>
+
+          <div style="gap: 1rem" class="flex flex-col">
+            <div
+              style="display: block"
+              class="btn btn-square-outlined btn-bg-hover-cyan-600 text-hover-white text-cyan-600"
+            >
+              HyperText Markup Language
+            </div>
+            <div
+              style="display: block"
+              class="btn btn-square-outlined btn-bg-hover-cyan-600 text-hover-white text-cyan-600"
+            >
+              HighText Markup Language
+            </div>
+            <div
+              style="display: block"
+              class="btn btn-square-outlined btn-bg-hover-cyan-600 text-hover-white text-cyan-600"
+            >
+              HyperText Markdown Language
+            </div>
+            <div
+              style="display: block"
+              c
```

Comment:
Please try to avoid inline styling.
There is a typo aswell on line 184 please fix that